dts/bindings/sensor/st,lis2dh-common.yaml

```diff
@@ -20,8 +19,24 @@ properties:
 
     anym-on-int1:
       type: boolean
-      required: false
```

Comment:
Did you mean to remove `required: false` for `anym-on-int1` and ` disconnect-sdo-sa0-pull-up`?

Reply:
Yes, they don't make any sense for `boolean` types, right?

Reply:
> Yes, they don't make any sense for boolean types, right?

That's a separate commit, then. Commits should do one thing. Please watch https://www.youtube.com/watch?v=mnVNLM7ynMQ